src/workers/webdav/modules/files/domain/WebdavFileRepository.ts

```diff
@@ -17,4 +17,6 @@ export interface WebdavFileRepository {
   searchOnFolder(
     folderId: WebdavFolderAttributes['id']
   ): Promise<Array<WebdavFile>>;
+
+  runRemoteSync(): Promise<void>;
```

Comment:
The repository interface should not know about the sync, that is part of the implementation